apps/percona/chaos/cstor_pool_scale/test.yml

```diff
@@ -0,0 +1,179 @@
+          args:
+            executable: /bin/bash
+          register: app_status
+          until: "((app_status.stdout_lines|unique)|length) == 1 and 'Running' in app_status.stdout"
+          delay: 10
+          retries: 12
+
+        - name: Get application pod name 
+          shell: >
+            kubectl get pods -n {{ namespace }} -l {{ label }} --no-headers
+            -o=custom-columns=NAME:".metadata.name"
+          args:
+            executable: /bin/bash
+          register: app_pod_name  
+
+        - name: Getting the number of cstor-pool pods
+          shell: kubectl get pods -n {{ operator_ns }} -l app=cstor-pool --no-headers | wc -l
+          register: pool_count
+          failed_when: pool_count.stdout|int < 3  
+
+        - name: Getting the volume name
+          shell: kubectl get pvc -n {{ namespace }} -o jsonpath='{.items[0].spec.volumeName}'
+          register: vol_name  
+
+        - name: Get CVR status list from pv
+          shell: >
+            kubectl get cvr -n {{ operator_ns }}
+            -l openebs.io/persistent-volume={{ vol_name.stdout }} --no-headers
+            -o jsonpath='{range .items[*]}{.status.phase}{"\n"}{end}'
+          args:
+            executable: /bin/bash
+          register: cvr_status_phase
+
+        - name: Check status of cvr
+          command: echo "{{ item }}"
+          failed_when: "item != \"Offline\" and item != \"Degraded\" and item != \"Rebuilding\" and item != \"Healthy\""
+          with_items:
+            - "{{ cvr_status_phase.stdout_lines }}"
+  
+        - name: Getting the pool deployment name
+          shell: kubectl get deployment -l app=cstor-pool -n {{ operator_ns }} --no-headers -o jsonpath='{.items[0].metadata.name}'
+          register: deployment_name
+
+        - name: Getting the cvr for the deployment
+          shell: kubectl get cvr -l cstorpool.openebs.io/name={{ deployment_name.stdout }} -n {{ operator_ns }} -o jsonpath='{.items[0].metadata.name}'
+          register: cvr_name1  
+
+        - name: Scaling-down cstor-pool
+          shell: kubectl patch deployment {{ deployment_name.stdout }} -n {{ operator_ns }} --type=json -p='[{"op":"add", "path":"/spec/replicas", "value":"0"}]'
+          register: result1
+          failed_when: "'patched' not in result1.stdout"
+
+        - name: Checking for pool scale-down
+          shell: kubectl get pods -n {{ operator_ns }} -l app=cstor-pool --no-headers | wc -l
+          register: downscale_value
+          until: "(downscale_value.stdout|int) == ((pool_count.stdout|int)-1)"
+          retries: 5
+          delay: 15   
+
+        - name: Create some test data in the mysql database
+          include_tasks: "/common/utils/mysql_data_persistence.yml"
+          vars:
+            status: 'LOAD'
+            ns: "{{ namespace }}"
+            pod_name: "{{ app_pod_name.stdout }}"
+            dbuser: 'root'
+            dbpassword: 'k8sDem0'
+            dbname: 'tdb'
```

Comment:
@shashank855 create unique string and pass as a dbname.

Reply:
@nsathyaseelan I have made the changes !!